### PR TITLE
 #162948878 get comment

### PR DESCRIPTION
**Pull request by jamesbeamie**
#### What does this PR do?
* Has implementation for users to view comments

#### Description of Task to be completed?
* Implement get comments for a specific article

#### How should this be manually tested?
* Clone the github repository `git clone https://github.com/andela/ah-alpha-frontend` on your local machine.
* run `git checkout ft-paginate-162948879`
* Install the required dependencies using `npm install`
* Run the local development server using `npm start`

#### Screenshot
image

#### What are the relevant pivotal tracker stories?
[#162948878](https://www.pivotaltracker.com/story/show/162948878)

### CODE REVIEW

File: src/components/containers/commentsContainer.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-shadow */
+// Container for all comments. be rendered into Articles app
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import PropTypes from "prop-types";
+
```
Comment (danuluma) [inline src/components/containers/commentsContainer.js]: Please remove this empty line

**Review by danuluma — APPROVED** [review]

**Review by Ruiru11 — APPROVED** [review]

File: src/actions/__test__/commentActions.test.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  it("should dispatch FETCH_COMMENTS_FAILURE on error when fetching comments", () => {
+    // List of comments
+    const errors = {};
+    const expectedAction = {
+      type: types.FETCH_COMMENTS_FAILURE,
+      payload: errors
+    };
+    expect(actions.fetchCommentsFailure(errors)).toEqual(expectedAction);
+  });
+});
```
Comment (tesh254) [inline src/actions/__test__/commentActions.test.js:26]: Always leave an extra line on each file its a coding convention

File: src/actions/__test__/getArticlesAction.test.js [hunk 3]
```diff
@@ -31,7 +31,7 @@ describe("actions", () => {
     return store.dispatch(getArticles()).then(() => {
       const dispathedActions = store.getActions();
 
-      expect(dispathedActions).toContainEqual({"isLoading": true, "type": "GET_ARTICLES_REQUEST"});
+      expect(dispathedActions).toContainEqual({ isLoading: true, type: "GET_ARTICLES_REQUEST" });
```
Comment (tesh254) [inline src/actions/__test__/getArticlesAction.test.js]: Format this with prettier it seems you have exceeded the amount of characters for each line of code which is 80-85

File: src/components/comment/commentsHere.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    const { comment } = this.props;
+    const { body, author_profile: { image, username } } = comment;
+    return (
+      <div className="ui comments">
+        <div className="comment">
+          <a className="avatar">
+            <img src={image} alt="commenter avatar" />
+          </a>
+          <div className="content">
+            <a className="author"><p>{username}</p></a>
```
Comment (tesh254) [inline src/components/comment/commentsHere.js]: Include a dummy href tag to avoid this Hound error

Comment (tesh254) [inline src/components/comment/commentsHere.js]: Introduce a href attribute to ignore this Hound linting errors

File: src/components/containers/articles/Articles.js [hunk 5]
```diff
@@ -51,7 +51,7 @@ export class GetArticles extends Component {
   render() {
     const pageNumbers = Math.ceil(this.state.count / this.state.page_size);
     return (
-      <div>
+      <div >
```
Comment (tesh254) [inline src/components/containers/articles/Articles.js]: Remove the space between

**Review by Deekerubo — APPROVED** [review]

File: src/actions/__test__/commentActions.test.js (hunk 2 above)
Comment (jamesbeamie) [reply src/actions/__test__/commentActions.test.js:26]: There is a line after this file

**Review by tesh254 — APPROVED** [review]

**Pull request merged**